src/datapoints-graph/charts/charts.component.ts

```diff
@@ -224,14 +479,34 @@ export class ChartsComponent implements OnChanges, OnInit, OnDestroy {
     });
   }
 
+  private async loadAlarmsAndEvents(): Promise<any> {
+    const timeRange = this.getTimeRange();
+    const filteredAlarmsOrEvents = this.config.alarmsEventsConfigs.filter(
+      (alarmOrEvent) => !alarmOrEvent.__hidden
+    );
```

Comment:
what about inactive a/e? seem like all of a/e are requested, even if not needed (because they have prop active: false)